### PR TITLE
Eb sprint 1 movies

### LINKED ISSUE #1: Sprint 0- Set up
- [x] Familiarize selves with project
- [x] Create Wireframe
- [x] Complete DTR
- [x] Set up Project Board 
- [x] Create PR Template

### LINKED ISSUE #2: Sprint 1 - Webpage
- [x] create basic HTML for webpage
- [x] Familiarize with API endpoints via Postman
- [x] get all movies API
- [x] set up project files structure

### PR DESCRIPTION
**Pull request by ericberglund117**
## Description

- Updated App.js file to include fetched movies data
- Create componentDidMount function to fetch all movies data
- Create separateMovies function to map over movies to display each movie's poster, title, rating, and release date
- Create render method to display retrieved movie data 

## Motivation and Context

- Had to fetched the data in order to display all the data on the homepage

## How Has This Been Tested?

- Through the browser. 
- did the data show? Yes it did

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [] All new and existing tests passed.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,49 @@
...
+
+  separateMovies() {
+    const moviesToDisplay = this.state.movies.map((movie, index)=> {
+      return (
+        <section key={index}>
+          <img src={movie.poster_path}></img>
+          <h2>{movie.title}</h2>
+          <h3>{movie.average_rating}</h3>
+          <h3>{movie.release_date}</h3>
+        </section>
```
Comment (MichaelEWalker87) [inline src/App.js:27]: might look at moving this out into another file in the future

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,49 @@
...
+    super();
+    this.state = {
+      movies: []
+    };
+  }
+
+  componentDidMount() {
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+    .then(response => response.json())
+    .then(data => this.setState({ movies: data.movies }));
```
Comment (MichaelEWalker87) [inline src/App.js:16]: In the next step, we are looking at adding error handling. 
We are still not sure if we want to split this into another file later

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,49 @@
...
+  }
+
+  separateMovies() {
+    const moviesToDisplay = this.state.movies.map((movie, index)=> {
+      return (
+        <section key={index}>
+          <img src={movie.poster_path}></img>
+          <h2>{movie.title}</h2>
+          <h3>{movie.average_rating}</h3>
+          <h3>{movie.release_date}</h3>
```
Comment (MichaelEWalker87) [inline src/App.js:26]: might need to add classes for targeting in SASS

**Review by MichaelEWalker87 — APPROVED** [review]
Great work overall I made a few small notes that we had already talked over.

File: src/App.js (hunk 1 above)
Comment (ericberglund117) [reply src/App.js:27]: Great idea, I think we should do that before we get too far

**Pull request merged**